cegs_portal/search/templates/search/v1/search_results.html

```diff
@@ -326,6 +493,22 @@
             updateDynamicIslands();
         });
 
+        state.addCallback(STATE_REGION, (s, key) => {
+            let region = state.g(STATE_REGION);
+            let featureTypes = state.g(STATE_FEATURE_TYPES);
+            let form = document.querySelector('form[hx-get]');
+            let url = `/search/v1/featureloc/chr${region.chr}/${region.start}/${region.end}`;
+
+            if (featureTypes.length > 0) {
+                url += `?feature_type=${featureTypes.join('&feature_type=')}`;
+            }
+
+            form.setAttribute('hx-get', url);
+
+            htmx.ajax('GET', url, {target: '#features-table'});
```

Comment:
These lines are exactly the same as 449-458, I think. This should be factored out into a function in the same way `updateDynamicIslands` was.